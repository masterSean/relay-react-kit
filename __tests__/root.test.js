import React, { Component } from "react";
import Root from "../src/component/root";
import renderer from "react-test-renderer";

test('root component testing', () => {
    const props = {
        hello: "Hello world"
    }
    const component = renderer.create(
        <Root greetings={props}/>  
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
