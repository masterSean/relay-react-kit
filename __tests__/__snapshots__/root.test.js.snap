exports[`test root component testing 1`] = `
<h1>
  Hello world
</h1>
`;
